Composer/packages/extensions/adaptive-form/src/PluginContext.ts

```diff
@@ -6,6 +6,7 @@ import { PluginConfig } from '@bfc/extension';
 
 const PluginContext = React.createContext<Required<PluginConfig>>({
   uiSchema: {},
+  visual: {},
```

Comment:
these names here are odd. `uiSchema` ? `visual` ? Questions I have as a developer using this interface:

1. What's the difference? `uiSchema` corresponds to forms and `visual` corresponds to the flow editor? Not very self-descriptive.
2. Why the need for both?

Reply:
What do you think would be most intuitive?

I think making a change like make sense.

```ts
interface uiSchema {
  formSchema: FormSchema;
  flowSchema: FlowSchema;
}
```

Reply:
LGTM. @yeze322 can you make this change?

Reply:
@a-b-r-o-w-n @cwhitten Great! It's a good name, I will update it

Reply:
`visual` renamed to `flowSchema`, renamed `uiSchema` to `formSchema` in a separated PR https://github.com/microsoft/BotFramework-Composer/pull/2634